ui/radar/radar.js

```diff
@@ -264,8 +270,14 @@ class Radar {
       const matches = log.match(NetRegexes.addedCombatantFull());
       if (matches) {
         const monster = this.nameToMonster[matches.groups.name.toLowerCase()];
-        if (monster)
+        if (monster) {
+          if (matches.groups.currentHp == 0) // don't fire if current HP = 0 (hunt is already dead)
```

Comment:
Can you make this `matches.groups.currentHp === '0'` since it's a string? Or alternatively `parseInt(matches.groups.currentHp) === 0`